config/config.go

```diff
@@ -168,6 +180,23 @@ func newDefaultMetricsConfig() *MetricsConfig {
 	}
 }
 
+type TraceConfig struct {
+	// JaegerEndpoint is the URL traces are collected on.
+	JaegerEndpoint string `json:"jaegerEndpoint"`
```

Comment:
unimportant nit: because of similarities with this and metrics it could be nice to have a consistent ordering among the three fields across both configs.